Sources/SwiftParser/Expressions.swift

```diff
@@ -525,6 +525,46 @@ extension Parser {
           arena: self.arena
         )
       )
+
+    case (.repeat, let handle)?:
+      // 'repeat' is the start of a pack expansion expression.
+      return RawExprSyntax(parsePackExpansionExpr(repeatHandle: handle, flavor, pattern: pattern))
```

Comment:
I just noticed that we already never parse `repeat` as a function call in the following (not a regression introduced by this PR, this just made me realize it…

```swift
func repeat() {}
repeat()
```

Reply:
Ah, thanks for reminding me. Now that you mention it, I think I left the same comment somewhere else already 🤦🏽